### PR TITLE
fix: fast username input bug fixed

### PR DESCRIPTION
**Pull request by amanuela97**
- username field in form is not longer buggy when typing fast

### CODE REVIEW

File: components/TField.vue [hunk 1]
```diff
@@ -33,7 +33,7 @@
           :hide-label="hideLabel"
           :item="item"
           v-on="$attrs.listeners"
-          @input="(val) => $emit('input', set(val))"
+          @update="(val) => $emit('input', set(val))"
```
Comment (razbakov) [inline components/TField.vue:36]: This will emit `input` event, but later on you use `update` event. Are you sure it works?

**Comment by razbakov** [comment]
Please add:

- link to netlify, where I can test it
- link the issue

**Comment by amanuela97** [comment]
[https://practical-visvesvaraya-ad7667.netlify.app/](url)
-link for testing username field

**Comment by razbakov** [comment]
I tried to sign up, filled all the field, but I am getting this error:

Screenshot 2021-08-09 at 11 47 32

File: components/TForm.vue [hunk 2]
```diff
@@ -157,8 +157,6 @@ export default {
       if (field && field.onChange) {
         field.onChange(val)
       }
-
-      this.$emit('input', val)
```
Comment (razbakov) [inline components/TForm.vue:161]: this is probably used somewhere, did you check?

Comment (amanuela97) [reply components/TForm.vue:161]: i did and form seems to function the same without it

Comment (razbakov) [reply components/TForm.vue:161]: check commit where this line was added and see why

Comment (razbakov) [reply components/TForm.vue:161]: It sets a new value for the form, whenever any field is changed. If this event is not emitted `v-model` of the form won't be updated.

File: components/TForm.vue [hunk 3]
```diff
@@ -136,16 +147,25 @@ export default {
 
       this.$emit('copy', this.value)
     },
-    save() {
+    async save() {
       this.error = false
 
       if (!this.validate()) {
```
Comment (razbakov) [inline components/TForm.vue:153]: You can put validation inside this function

**Comment by razbakov** [comment]
fixed in #50

**Pull request closed without merging**